documentation/openapi-authoring-automated-guidelines.md

```diff
@@ -1015,13 +1016,13 @@ Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rul
 
 Links: [Index](#index) | [Error vs. Warning](#error-vs-warning) | [Automated Rules](#automated-rules) | [RPC](#rpc-violations): [Errors](#rpc-errors) or [Warnings](#rpc-warnings) | [SDK](#sdk-violations): [Errors](#sdk-errors) or [Warnings](#sdk-warnings)
 
-### <a name="R2064" />R2064 LicenseMissing
-**Output Message**: Please provide correct licensing information here. Acceptable value: "name": "MICROSOFT\_MIT\_NO\_VERSION"
+### <a name="R2065" />R2065 LicenseHeaderMustNotBeSpecified
+**Output Message**: License header must not be specified inside x-ms-code-generation settings. This is different for different sdks generated and is passed via command line/config file when generating the sdk.
 
-**Description**: Each OpenAPI json document must contain the license object in its `info` section which must set the name to `MICROSOFT\_MIT\_NO\_VERSION`. This is necessary for AutoRest to insert the appropriate headers in the generated code.
+**Description**: `x-ms-code-generation-settings` must not have the license section specified in the OpenAPI documents since each SDK has a different licensing header, this information must be provided either from the command line or the configuration file when actually generating the sdk.
```

Comment:
"Output Message" and "Description" are identical modulo spelling and grammar used. If there is really not more to say in the description than in the output message, just copy paste the output message, I see no value in having identical information in similar rather than identical, but high quality words. 😉